spring-cloud-gateway-server/src/main/java/org/springframework/cloud/gateway/route/builder/GatewayFilterSpec.java

```diff
@@ -447,6 +449,27 @@ public GatewayFilterSpec redirect(HttpStatus status, URL url) {
 		}
 	}
 
+	/**
+	 * A filter that can be used to modify the response body.
+	 * @param attributes list of attributes to remove separated by commas, an optional
+	 * last parameter from the list can be a boolean to remove the attributes just at root
+	 * level (false) o recursively (true)
+	 * @return a {@link GatewayFilterSpec} that can be used to apply additional filters
+	 */
+	public GatewayFilterSpec removeJsonAttributes(String... attributes) {
```

Comment:
create an overload with first parameter as the boolean. The java dsl shouldn't rely on parsing a string.

Reply:
Included as parameter: https://github.com/spring-cloud/spring-cloud-gateway/pull/2742/commits/bcd4eeb4979f19539f4da656161826741f86d3c7